game/menu/README.md

```diff
@@ -0,0 +1,39 @@
+# Info for Contributers
+
+## GameManager
+The Gamemanager keeps track of all minigames and starts/stops them when needed.
+It also has a rudimentary game selection menu that uses the Gamedisplays.
+
+## Gamedisplay
+Is used my the Gamemanagers game menu to show what the game is about.
+It has a button to load the game & a popup for more detailed infos but no logic beyond that.
+
+
+# API
+## GameManager
+
+* **`load_game(game_cfg:ConfigFile)`** <br>
```

Comment:
I'd always add a space character after `:` for better readability and following typical typography rules.